readit/database.py

```diff
@@ -295,6 +286,8 @@ def export_urls(self):
         try:
             self.conn = sqlite3.connect(glob(os.path.expanduser(databasefile))[0])
             self.cursor = self.conn.cursor()
+=======
+>>>>>>> 4725fbb... Code optimization in database.py and in view.py
             self.cursor.execute("select * from bookmarks")
```

Comment:
I think you missed this while resolving conflicts.

Reply:
I can still able to reproduce this issue with python3.7.5
I created new issue for it - https://github.com/pythonpune/readit/issues/159